lib/component.js

```diff
@@ -161,6 +161,21 @@ class Component extends WebComponent {
     }
   }
 
+  /**
+   * Returns the reference to the parent of the given name. An error is thrown out if no parent of
+   * such name is found.
+   * @param {string} tagName - name of the parent to search for.
+   * @example this.parentbyTagName('my-app');
+   */
+  parentByTagName(tagName) {
```

Comment:
let's add `tagName = tagName.toLowerCase()` at the top of the method so the caller doesn't have to worry about case at all.